pootle/apps/pootle_fs/matcher.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import logging
+import os
+from fnmatch import fnmatch
+
+
+logger = logging.getLogger(__name__)
+
+
+class FSPathMatcher(object):
+
+    def __init__(self, context):
+        self.context = context
+
+    @property
+    def lang_mapper(self):
+        return self.context.lang_mapper
+
+    @property
+    def project(self):
+        return self.context.project
+
+    @property
+    def root_directory(self):
+        return self.context.project.local_fs_path
+
+    @property
+    def translation_path(self):
+        return self.context.project.config[
+            "pootle_fs.translation_paths"]["default"]
```

Comment:
in the original implementation there was an additional method, this iterated through the translation_path matches using a slightly iffy (but maybe ok) algorithm. This was what i removed.

So now we still have a `translation_paths` dictionary, im thinking we might want to just replace that with a `translation_path` string